src/System.Net.HttpListener/src/Resources/Strings.resx

```diff
@@ -313,28 +372,7 @@
   <data name="SSPIInvalidHandleType" xml:space="preserve">
     <value>'{0}' is not a supported handle type.</value>
   </data>
-  <data name="offset_out_of_range" xml:space="preserve">
-    <value>Offset exceeds the length of buffer.</value>
-  </data>
-  <data name="net_io_operation_aborted" xml:space="preserve">
-    <value>I/O operation aborted: '{0}'.</value>
-  </data>
-  <data name="net_invalid_path" xml:space="preserve">
-    <value>Invalid path.</value>
-  </data>
-  <data name="net_no_client_certificate" xml:space="preserve">
-    <value>Client certificate not found.</value>
-  </data>
-  <data name="net_listener_auth_errors" xml:space="preserve">
-    <value>Authentication errors.</value>
-  </data>
-  <data name="net_listener_close" xml:space="preserve">
-    <value>Listener closed.</value>
-  </data>
-  <data name="net_invalid_port" xml:space="preserve">
-    <value>Invalid port in prefix.</value>
-  </data>
   <data name="net_WebSockets_InvalidState" xml:space="preserve">
     <value>The WebSocket is in an invalid state ('{0}') for this operation. Valid states are: '{1}'</value>
   </data>
-</root>
+</root>
```

Comment:
nit: missing newline at end of file.

Reply:
Is that newline really important? Because I do not remove it intentionally. I just edit resources in Visual Studio and this file is automatically generated without the last newline.

Personally I also like it and add new line after the last line in the files, but not in generated files.